samples/net/wpanusb/src/wpanusb.c

```diff
@@ -4,8 +4,7 @@
  * SPDX-License-Identifier: Apache-2.0
```

Comment:
The commit message tells the same and obvious thing as what can be seen in diff. What I would like to see here is the reason/motivation why this change is needed.

Reply:
I think this is mostly USB code (maybe should be moved to USB samples), If I want to set log level I can do this in "USB Device Log Level" menu, how it could be done with NET_* macros?

Reply:
Sure, I am ok if you move it to usb samples instead. I am also ok with USB_DBG() etc, I was just wondering why this change is needed.